x-pack/legacy/plugins/siem/public/pages/case/components/tag_list/index.tsx

```diff
@@ -106,14 +98,18 @@ export const TagList = React.memo(({ tags, isEditTags, iconAction }: TagListProp
               </Form>
             </EuiFlexItem>
             <EuiFlexItem>
-              <EuiButton fill onClick={onSubmit}>
-                {i18n.SUBMIT}
-              </EuiButton>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiButtonEmpty onClick={() => onActionClick(iconAction.onClick, false)}>
-                {i18n.CANCEL}
-              </EuiButtonEmpty>
+              <EuiFlexGroup gutterSize="s" alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiButton color="secondary" fill iconType="save" onClick={onSubmitTags} size="s">
+                    {i18n.SAVE}
+                  </EuiButton>
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiButtonEmpty iconType="cross" onClick={() => setIsEditTags(false)} size="s">
```

Comment:
nit -> you can use `useCallback` or 
```suggestion
                  <EuiButtonEmpty iconType="cross" onClick={setIsEditTags.bind(null, false)} size="s">
```